docs/guides/admin/docs/configuration/multi.tenancy.md

```diff
@@ -75,7 +76,7 @@ do not support multitenancy.
 Note that if you are running Apache httpd with mod\_proxy in front of the Opencast installation, the port number will be
```

Comment:
Line 70 also still references the old `server` key that no longer exists. (Unfortunately I can't comment on that line directly... whyyy, Github!)

Instead this should be probably:

    prop.org.opencastproject.host.admin.opencast.org=admin.opencast.org
    prop.org.opencastproject.host.presentation.opencast.org=presentation.opencast.org

Reply:
Or rather *.example.org